server/fleet/activities.go

```diff
@@ -491,7 +491,7 @@ type Activity struct {
 	ActorFullName *string          `json:"actor_full_name,omitempty" db:"name"`
 	ActorID       *uint            `json:"actor_id,omitempty" db:"user_id"`
 	ActorGravatar *string          `json:"actor_gravatar,omitempty" db:"gravatar_url"`
-	ActorEmail    *string          `json:"actor_email,omitempty" db:"email"`
+	ActorEmail    *string          `json:"actor_email,omitempty" db:"user_email"`
```

Comment:
Was this a bug? Previous `email` metadata was useless?

Reply:
No, it wasn't a bug before, it was just coming from the `users` table. The thing that changes with this PR is that this `user_email` column now exists as a fallback in case the `user` record is deleted.